_data/sidebars/contrib_sidebar.yml

```diff
@@ -39,7 +39,12 @@ children:
   - title: LoopBack 4
     url: code-contrib-lb4.html
     output: web, pdf
+    children:
 
+    - title: Submitting a pull request to LoopBack 4
+      url: /doc/en/lb4/submitting_a_pr.html
```

Comment:
Since this is under the `contrib_sidebar`, shouldn't the file be in `pages/en/contrib`?

Reply:
@nabdelgadir 
This goes back to my initial document. I put the document and sidebar changes in loopback.io ...originally.

But then I was asked to put the document in loopback-next since this was a LoopBack 4 tutorial.

Now, we need to link to it from a sidebar YML and document located in loopback.io.

So... that's why the changes are in 2 places.